module/VuFind/src/VuFind/Navigation/AccountMenuFactory.php

```diff
@@ -65,11 +65,9 @@ public function __invoke(
         ?array $options = null
     ) {
         // Only load the connector if we need to show
-        $config = $container->get(\VuFind\Config\PluginManager::class)->get(
-            'Overdrive'
-        );
+        $config = $container->get(\VuFind\Config\ConfigManager::class)->getConfigArray('Overdrive');
         $connector = null;
-        if ($config->Overdrive->showMyContent != 'never') {
+        if ($config['Overdrive']['showMyContent'] != 'never') {
```

Comment:
Maybe we need a fallback here:

```suggestion
        if (($config['Overdrive']['showMyContent'] ?? '') != 'never') {
```